pkg/migrate/context.go

```diff
@@ -0,0 +1,11 @@
+package migrate
+
+// MigrationVariable contains constants that can be used as context keys that might
+// be relevant in a number of different migration scenarios.
+type MigrationVariable int
```

Comment:
do we want to make these ints vs strings? I have concerns there might be accidental overlap with something we don't inject

Reply:
Not possible, the equality operator the context uses considers the type:
https://go.dev/play/p/16BfmcC0d9X